assets/content/less/buttons/buttons.less

```diff
@@ -0,0 +1,73 @@
+// lesshint qualifyingElement: { enabled: true, allowWithClass: true, allowWithAttribute: true }
+.btn {
+    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    font-size: 14px;
+    padding-left: 29px;
+    padding-right: 31px;
+
+    &:hover {
+        box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.23);
+    }
+
+    // Blue button
+    &.btn-primary,
+    &.btn-primary:hover {
+        background: #65a6de;
```

Comment:
Bootstrap is being included currently on line 27: https://github.com/crowbar/crowbar-ui/blob/master/bower.json#L27

But we're not consuming it importing the less file, but the final .min files (https://github.com/crowbar/crowbar-ui/blob/master/gulp/bowerExtract.js#L51), so I don't think variables replacement is that easy now. :/

But we could research and improve that in next sprint and track the variables replacement there, as its out of the current PR's scope.